resources/creatives/models/creative.yml

```
@@ -30,35 +30,85 @@ allOf:
       template_preview_urls:
         title: Template Preview URLs
         type: object
-        description: URL to view preview of a creative ID. Currently unpopulated, but will be in the future.
+        description: URL to view preview of a creative ID. Currently unpopulated, but will be filled in the future.
       template_previews:
         title: Template Previews
         type: array
         items:
           type: object
-        description: IDs for each creative preview. Currently unpopulated, but will be in the future.
+        description: IDs for each creative preview. Currently unpopulated, but will be filled in the future.
```

Another suggested desc:
```suggestion
A list of template preview objects if the creative uses HTML template(s) as artwork asset(s). An empty array will be returned if no `template_preview`s have been generated for the creative.
```